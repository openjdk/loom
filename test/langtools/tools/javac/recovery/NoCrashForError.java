/**
 * @test /nodynamiccopyright/
 * @bug 8286895
 * @summary Verify that errors don't crash the compiler.
 * @compile/fail/ref=NoCrashForError.out -XDshould-stop.at=FLOW -XDdev -XDrawDiagnostics NoCrashForError.java
 */
public class NoCrashForError {
    private void JDK8286895() {
        Number n = 17;
        if (! n instanceof Integer i) {
            System.out.println("not Integer");
        }
    }
}
