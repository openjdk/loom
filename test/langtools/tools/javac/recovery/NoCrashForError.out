NoCrashForError.java:10:13: compiler.err.operator.cant.be.applied: !, java.lang.Number
1 error
